Resources/Prototypes/Loadouts/Jobs/Engineering/seniorEngineer.yml

```
@@ -412,6 +412,18 @@
   items:
     - SheetPlastic10
 
+- type: loadout
+  id: LoadoutSeniorEngineerEquipmentExperimentalWeldingTool
+  category: JobsEngineeringSeniorEngineer
+  cost: 0
+  requirements: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)